src/Modules/Backend/assets/Backend/webpack/scss/components/_tables.scss

```diff
@@ -7,12 +7,12 @@
   border: 1px solid $table-border-color;
 }
 
-.table > tbody > tr > td.dragAndDropHandle,
-.table > tbody > tr > th.dragAndDropHandle,
-.table > thead > tr > th.dragAndDropHandle,
-.table > tbody > tr > td.sortHandle,
-.table > tbody > tr > th.sortHandle,
-.table > thead > tr > th.sortHandle {
+.table > tbody > tr > td.drag-and-drop-handle,
```

Comment:
@carakas I have a task for @daphneslootmans to cleanup CSS and JS when fork6 is a bit further. 
Let's focus here on the CI issue, otherwise this PR will get out of focus.